search/src/main/java/edu/artic/search/SearchResultsAdapter.kt

```diff
@@ -76,6 +77,7 @@ class SearchResultsAdapter : AutoHolderRecyclerViewAdapter<SearchBaseCellViewMod
 
                     }
                     is SearchTourCellViewModel -> {
```

Comment:
Please remove the empty `is` branches and add an `else` block setting `itemSubTitle` to `VISIBLE`.

Reply:
Updated.